react-multiple-pages/my-app/src/App.tsx

```diff
@@ -0,0 +1,23 @@
+import { About } from './pages/About';
+import { Dashboard } from './pages/Dashboard';
+import { Header } from './components/Header';
+import { NotFound } from './pages/NotFound';
+import { Details } from './pages/Details';
+import { Routes, Route } from 'react-router-dom';
+
+export function App() {
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<Header />}>
+          <Route index element={<Dashboard />} />
+          <Route path="/about" element={<About />} />
+          <Route path="/people">
```

Comment:
You can combine these two into `path="/people/:itemId"`

Usually you would name the path element "person" since it is taking you to a page that describes a single person.